src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -420,7 +434,6 @@ They can be configured through the settings file 'winget settings'.</value>
   </data>
   <data name="Package" xml:space="preserve">
     <value>Package</value>
-    <comment>A software package</comment>

```

Comment:
><comment>A software package</comment> [](http://example.com/codeflow?start=4&length=37)

Keep this comment to avoid translator translating to other "package"